.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
```

Comment:
Curious: is there a reason why `xenial` is specifically specified here instead of `trusty` or just left blank?